modules/migration-about-cpma.adoc

```diff
@@ -1,9 +1,9 @@
 // Module included in the following assemblies:
-// migration/migrating_3-4/migrating-application-workloads-3-to-4.adoc
-[id='migration-understanding-cpma_{context}']
-= Understanding the Control Plane Migration Assistant
+// * migration/migrating_3_4/migration-concepts-prerequisites-3-4.adoc
```

Comment:
Same in this file about reverting the file rename and anchor, and fixing the assembly list.